src/components/GoodsList/GoodsList.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const GoodsList = ({ goodsList }) => {
+  if (goodsList.length === 0) {
+    return (
+      <p>Please press the load button</p>
+    );
+  }
```

Comment:
But what if there are no goods in the list?
User will go crazy pressing load button 😆